src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
```

Comment:
The superclass is "thinner" than most :-). Question for you: should "isValid()" be a superclass method? Why?